system-parachains/constants/src/lib.rs

```diff
@@ -57,3 +57,42 @@ pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 
 /// Treasury pallet id of the local chain, used to convert into AccountId
 pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"py/trsry");
+
+/// Parameters supporting async backing functionality.
+///
+/// Once all system chains have migrated to the new async backing mechanism, the parameters
+/// in this namespace will replace those currently defined in `super::*`.
+pub mod async_backing {
+	use frame_support::weights::{constants::WEIGHT_REF_TIME_PER_SECOND, Weight};
+	pub use parachains_common::BlockNumber;
+	use sp_runtime::Perbill;
+
+	/// This determines the average expected block time that we are targeting.
```

Comment:
the comment is not really accurate anymore. I don't see `MILLISECS_PER_BLOCK` being used anywhere for determining the block time